h2o-docs/src/product/automl.rst

```diff
@@ -68,8 +68,13 @@ Optional Miscellaneous Parameters
 
 - `max_runtime_secs_per_model <data-science/algo-params/max_runtime_secs_per_model.html>`__: Specify the max amount of time dedicated to the training of each individual model in the AutoML run. Defaults to 0 (disabled). Note that setting this parameter can affect AutoML reproducibility.
 
+- **modeling_plan**: The list of modeling steps to be used by the AutoML engine (they may not all get executed, depending on other constraints).
+
+- `monotone_constraints <data-science/algo-params/monotone_constraints.html>`__: A mapping that represents monotonic constraints. Use +1 to enforce an increasing constraint and -1 to specify a decreasing constraint. Note that constraints can only be defined for numerical columns. This option can only be used when the distribution is either gaussian or bernoulli.
+
 -  `stopping_metric <data-science/algo-params/stopping_metric.html>`__: Specify the metric to use for early stopping. Defaults to ``AUTO``. The available options are:
     
+    - ``aucpr``
```

Comment:
even if not the first lexicographically, shouldn't we leave `AUTO` on top of the list?

Reply:
same below